test/intern/mixins/Tree.js

```diff
@@ -135,15 +139,62 @@ define([
 	}
 
 	test.suite('Tree', function () {
-		test.suite('large family expansion', function () {
 
-			test.beforeEach(function () {
-				createGrid();
+		function makeBeforeEach(setStoreAfterStartup) {
+			return function () {
+				createGrid({}, setStoreAfterStartup);
 
 				// Firefox in particular seems to skip transitions sometimes
 				// if we don't wait a bit after creating and placing the grid
 				return wait();
+			};
+		}
+
+		test.suite('no store', function () {
+
+			test.afterEach(destroyGrid);
+
+			test.test('call expand', function () {
+				grid = new (declare([OnDemandGrid, Tree]))(
+					{
+						columns: [
+							{ renderExpando: true, label: 'id', field: 'id' },
+							{ label: 'value', field: 'value' }
+						]
+					}
+				);
+				document.body.appendChild(grid.domNode);
+				grid.startup();
+
+				grid.expand(0);
```

Comment:
This doesn't seem like a valid test since there's nothing to even expand.  (Is this also why you added the `target &&` guard in the mixin?)